src/service/cid-impl.js

```diff
@@ -113,13 +113,15 @@ export class Cid {
      * Cached base cid once read from storage to avoid repeated
      * reads.
      * @private {?Promise<string>}
+     * @restricted
```

Comment:
curious what does this `@restricted` annotation do

Reply:
Means it may be used outside the class.